lib/maze/internal/fixture_config.rb

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+module Maze
+  module Internal
+    # Handles the Maze Runner configuration file for test fixtures
+    class FixtureConfig
```

Comment:
Do we envisage adding many more things here?

I'm wondering if this should just be a hash in `base_client`, or if it should be more aware of what's being put in it, e.g. rather than a generic `add` method, should there be a specific method for `maze_address`?

Reply:
It's a good question.  Initially I liked having it as its own specific abstraction, but it's over-engineered if we never need to add anything else - and in general we have commands to convey information to the fixture.  For now I'll just write a JSON string inline, saving the need for it completely.  We can factor it out again later if really necessary.